stdlib/public/core/Policy.swift

```diff
@@ -15,8 +15,26 @@
 //===----------------------------------------------------------------------===//
 // Standardized uninhabited type
 //===----------------------------------------------------------------------===//
-/// The return type of functions that do not return normally, that is, a type
-/// with no values.
+/// The return type of functions that cannot return a value.
```

Comment:
*The Swift Programming Language* book calls them "[nonreturning][]" functions.

[nonreturning]: <https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID551>